Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -245,7 +246,10 @@ extension BrowserViewController: WKNavigationDelegate {
         info["url"] = webView.URL
         info["title"] = webView.title
 
-        notificationCenter.postNotificationName("LocationChange", object: self, userInfo: info)
+        webView.runScript("getAll()", js: "Favicons", callback: { obj in
```

Comment:
If we do this after every location change then I think we should inject it via a `UserScript` and move it to a `BrowserHelper` implementation like `ReaderMode` is. Then it will automatically and more quickly be injected in web content.